alchemy-web/docs/providers/supabase/bucket.md

```diff
@@ -0,0 +1,207 @@
+
+The bucket resource exposes the following properties:
+
+- **`id`** (`string`): Unique identifier for the bucket
+- **`name`** (`string`): Bucket name
+- **`owner`** (`string`): ID of the bucket owner
+- **`public`** (`boolean`): Whether the bucket allows public access
+- **`createdAt`** (`string`): ISO timestamp when the bucket was created
+- **`updatedAt`** (`string`): ISO timestamp when the bucket was last updated
+
+## Examples
+
+### Basic Private Bucket
+
+```typescript
+const privateBucket = Bucket("user-documents", {
+  projectRef: "my-project-ref",
+  public: false,
+});
+```
+
+### Public Bucket with File Restrictions
+
+```typescript
+const imageBucket = Bucket("public-images", {
+  projectRef: "my-project-ref",
+  public: true,
+  fileSizeLimit: 10485760, // 10MB
+  allowedMimeTypes: [
+    "image/jpeg",
+    "image/png", 
+    "image/gif",
+    "image/webp"
+  ],
+});
+```
+
+### Bucket with Adoption
+
+```typescript
+// This will adopt an existing bucket if one with the same name already exists
+const existingBucket = Bucket("existing-bucket", {
+  projectRef: "my-project-ref",
+  adopt: true,
+  public: true,
+});
+```
+
+### Bucket that Won't be Deleted
+
+```typescript
+const persistentBucket = Bucket("persistent-storage", {
+  projectRef: "my-project-ref",
+  delete: false, // Bucket will not be deleted when resource is destroyed
+  public: false,
+});
+```
+
+### Large File Bucket
+
+```typescript
+const videosBucket = Bucket("video-uploads", {
+  projectRef: "my-project-ref",
+  public: false,
+  fileSizeLimit: 104857600, // 100MB
+  allowedMimeTypes: [
+    "video/mp4",
+    "video/mpeg",
+    "video/quicktime",
+    "video/webm"
+  ],
+});
+```
+
+## API Operations
+
+### Create Bucket
+- **Endpoint**: `POST /projects/{projectRef}/storage/buckets`
+- **Body**: Bucket configuration including name, public, file_size_limit, allowed_mime_types
+- **Response**: Bucket object with ID and configuration
+
+### List Buckets
+- **Endpoint**: `GET /projects/{projectRef}/storage/buckets`
+- **Response**: Array of bucket objects
+
+### Delete Bucket
+- **Endpoint**: `DELETE /projects/{projectRef}/storage/buckets/{name}`
+- **Response**: 200 on successful deletion
+
+## Error Handling
+
+The resource handles the following error scenarios:
+
+- **409 Conflict**: When `adopt: true` is set, the resource will attempt to find and adopt an existing bucket with the same name
+- **Rate Limiting**: Automatic exponential backoff for 429 responses
+- **Server Errors**: Automatic retry for 5xx responses
+- **404 on Delete**: Ignored (bucket already deleted)
+
+## Lifecycle Management
+
+- **Creation**: Buckets are created with the specified access controls and file restrictions
+- **Updates**: Bucket configuration can be refreshed to get current state
+- **Deletion**: Buckets can be deleted unless `delete: false` is specified
+
+## Dependencies
+
+Buckets depend on:
+- **Project**: Must specify a valid `projectRef`
+
+## Access Control
```

Comment:
Get rid of this whole dependencies section. Just focus on usage examples. Clean this up for all other resources too. Look at the Cloudflare alchemy docs in alchemy-web and follow those examples